### ISSUE TITLE
Lag. Lots of it.

### ISSUE THREAD

**Issue by MagmaBro123** [Gaelmare/dynamictreestfc#47]
Related to #22. While generating chunks, the FPS is very bad.

**Comment by Gaelmare** [comment]
Yes. Recommend pre-gen of chunks whenever you use DT. I believe this is not a Dynamic Trees TFC issue, but is endemic to DT in general.

**Comment by Gaelmare** [comment]
This is separate from #22, which is about cascading lag which happens when trees in loaded chunks grow into unloaded chunks.

**Comment by MagmaBro123** [comment]
> Yes. Recommend pre-gen of chunks whenever you use DT. I believe this is not a Dynamic Trees TFC issue, but is endemic to DT in general.

In a normal world DT doesn't cause this.
